frontend/public/src/containers/pages/checkout/OrderHistory.js

```diff
@@ -144,30 +147,31 @@ export class OrderHistory extends React.Component<Props> {
                 <h1 className="flex-grow-1">Order History</h1>
               </div>
             </div>
-            <div className="col-12 d-flex bg-light justify-content-between">
-              <table className="order-history-table">
-                <thead className="border-bottom border-2 border-dark">
-                  <tr>{columns}</tr>
-                </thead>
-
-                <tbody>
-                  {orderHistory && orderHistory.results.length > 0
-                    ? orderHistory.results.map(this.renderOrderCard.bind(this))
-                    : null}
-                  <tr className="border-top border-2 border-dark">
-                    <td>
-                      Page {this.offset + 1} of{" "}
-                      {orderHistory ? Math.ceil(orderHistory.count / 10) : 0} |{" "}
-                      {orderHistory ? orderHistory.count : "0"} orders total
-                    </td>
-                    <td /> <td /> <td />
-                    <td className="text-right">
-                      {this.renderPaginationPrevious()} |{" "}
-                      {this.renderPaginationNext()}
-                    </td>
-                  </tr>
-                </tbody>
-              </table>
+            <div className="row">
+              <div className="col-12 d-flex bg-light justify-content-between">
+                <table title="Order History" className="order-history-table">
+                  <thead className="border-bottom border-2 border-dark">
+                    <tr>{columns}</tr>
+                  </thead>
+                  <tbody>
+                    {orderHistory && orderHistory.results.length > 0
+                      ? orderHistory.results.map(this.renderOrderCard.bind(this))
+                      : null}
+                    <tr className="border-top border-2 border-dark">
+                      <td>
+                        Page {this.offset + 1} of{" "}
+                        {orderHistory ? Math.ceil(orderHistory.count / 10) : 0} |{" "}
+                        {orderHistory ? orderHistory.count : "0"} orders total
+                      </td>
+                      <td /> <td /> <td />
```

Comment:
I think these are causing an error to be thrown to the console in Google Chrome: `Whitespace text nodes cannot appear as a child of <tr>. Make sure you don't have any extra whitespace between tags on each line of your source code.`